.golangci.yml

```diff
@@ -0,0 +1,90 @@
+version: "2"
+linters:
+  enable:
+    # Commented entries here are not enabled by default, but that we should
+    # add once the relevant lines are fixed up.
+    
+    # - errorlint # found 2 cases to be fixed.
+    # - exhaustive # found some cases that should be fixed.
+    # - exhaustruct # found some structs which missed fields in initializing.
+    - govet
+    # - inamedparam  # fix missing named param
+    # - makezero  # fix the unallocated elements.
+    # - misspell  # fix spelling
+    # - nlreturn  # fix these missing blank line
+    # - nonamedreturns  # found some shadowing
+    # - predeclared  # found some shadowing
+    # - revive  # detects a number of good fixes
+    # - thelper  # fix test methods with t.Testing not as first param
+    - unconvert
+    # - wastedassign  # fix and enable
+    # - whitespacew  # fix and enable
+  disable:
+    # A number of these checks run by default so they are explicitly
```

Comment:
This isn't matching  http://golangci-lint.run/usage/linters.   As I mentioned in my previous comment, only these are enabled by default and firing:
```
- errcheck
- staticcheck
- unused
```

Is it possible that when you run `golangci-lint linters` it is pulling a config from somewhere else?

Reply:
I added every linter then enabled/disabled one by one to see which ones fire, how often, and if they were probably useful to figure out which to put in which list.  All the other linters are disabled by default and not included in the resulting config.

Reply:
Got it.  That makes sense, sorry for not reading the comment more closely.  For the ones where you're thinking about running them by hand occasionally, you could see if the only-new-issues option helps.